support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -417,9 +417,12 @@ AccessControlled accessControlled() {
                         cookie = UUID.randomUUID().toString();
                         // Switches the label to a self-label, so if the executable is killed and restarted via ExecutorPickle, it will run on the same node:
                         label = computer.getName();
+
```

Comment:
Workflow is a new plugin that adds ~~pain~~ to all other jenkins plugins, so other developers must have ability easily track logic changes. PR combines one change, but this contains unrelated to logic coding changes.
1) You can configure your IDE to not perform coding style changes for NOT changed parts of file (IDEA support it by default AFAIR)
2) Please do coding style changes in separate PRs
3) Please squash commits before merge for such small changes, i started reviewing changes in master and end in this commit, then realised that here just a https://github.com/jenkinsci/workflow-plugin/pull/145/commits 10 commits for one file + one block of code.
Thanks.